### DISCUSSION TITLE
Webpack: Still seeing cryptic errors after compiling corrections and js modularization

### DISCUSSION THREAD

**Discussion by GonzalesMatthew** [nss-evening-cohort-14/evening-client-side#105] [Help Me!] [answered]
Matthew Gonzales
Have you searched the other help tickets? Yes
Describe your issue:

I finished the compiling and js modularization steps. But still seeing these two errors in my console

DevTools failed to load SourceMap: Could not load content for chrome-extension://liecbddmkiiihnedobmlmillhodjkdmb/js/content.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME

DevTools failed to load SourceMap: Could not load content for chrome-extension://liecbddmkiiihnedobmlmillhodjkdmb/js/intercom-link-expand-loader.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME

List any error messages that appear:
DevTools failed to load SourceMap: Could not load content for chrome-extension://liecbddmkiiihnedobmlmillhodjkdmb/js/content.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME

DevTools failed to load SourceMap: Could not load content for chrome-extension://liecbddmkiiihnedobmlmillhodjkdmb/js/intercom-link-expand-loader.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME

What have you googled? 
I tried looking in my webpack repo for "intercom" and "content" keywords anywhere in the code but nothing. Also Google didn't really help. Just found ways to disable sourcemaps for JS and CSS but I don't think I want to do that.

What have you tried? 
Looked for hints in the webpack repo files but there are a lot!

Copy/paste link TO THE FILE on your branch or in your repo:
Not sure if it's a file or what but this is my repo path
https://github.com/GonzalesMatthew/sorting-hat-webpack

**Comment by ajawashington** [comment, marked as answer]
Those are just warnings that you don't have to worry about! ✨

**Comment by drteresavasquez** [comment]
@GonzalesMatthew we still need the links that you googled so that we can tell you they actually could have helped. Make sure to always provide the links and details. They may actually lead to good answers.
